README.md

```diff
@@ -9,22 +9,22 @@ You can [download Impact][download] by visiting the Impact Website.
 
 ## Help and Support
 
-If you need help using Impact, you should instead ask for it on the [discord] server. Use the `#help` channel so that you don't clutter up `#general` with support questions. Also make sure to check `#announcements`, pinned messages and use the search tool to see if your question has already been answered.
+If you need help using Impact, you should instead ask for it on the [discord] server. Use the `#help` channel so that you don't clutter up `#general` with support questions. Make sure to check `#announcements`, pinned messages and use the search tool to see if your question has already been answered.
```

Comment:
there was nothing wrong with this

Reply:
dont care didnt ask 😎

Reply:
i like don't care either way so im going to click RESOLVE CONVERSATION

troll face emoji sunglasses emoji cat smirk emoji